doc/api/deprecations.md

```diff
@@ -94,14 +94,6 @@ Within the [`child_process`][] module's `spawn()`, `fork()`, and `exec()`
 methods, the `options.customFds` option is deprecated. The `options.stdio`
 option should be used instead.
 
-<a id="DEP0007"></a>
-### DEP0007: cluster worker.suicide
```

Comment:
Yeah, it's more or less policy I think to leave the code?

Maybe we can just say "the predecessor to cluster worker.exitedAfterDisconnect"?